docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -11,40 +11,45 @@ Additionally, it utilizes the [Channels toolkit](/api-docs/channels/guide/overvi
 
 ## Fork and install the source repository 
 
+As a first step you need to fork the source repository, clone it to your development, and install the dpackages for your app
+
+**To fork and install the source repository**
+
 1. Fork the [subscription-foundation repository (GitHub)](https://github.com/bigcommerce/subscription-foundation) to your GitHub account.
 2. Clone your fork to your local development environment.
-3. Navigate to your clone's root directory and install the application's default packages with npm.
+3. Navigate to your root directory of your cloned repository and install the default packages for your app by issuing the following command:
 
 ```shell title="Install packages"
 npm install
 ```
 ## Create an HTTPS tunnel
 
-You need to serve your app over fully-qualified publicly accessible URL to connect the app to your sandbox store. We recommend using [ngrok](https://ngrok.com/docs), a free tool that uses HTTP tunneling to securely expose `localhost` ports to the public internet. 
+After you have successfully installed the packages for your app, need to serve your app over a fully-qualified publicly accessible URL to connect the app to your sandbox store. We recommend using [ngrok](https://ngrok.com/docs), a free tool that uses HTTP tunneling to securely expose `localhost` ports to the public internet. 
 
-Consult [our App Tutorial](/api-docs/apps/tutorials/sample-app-nextjs/step-2-connect#create-an-https-tunnel) to get started.
+To get started with ngrok, read the [Create an HTTPS Tunnel](/api-docs/apps/tutorials/sample-app-nextjs/step-2-connect#create-an-https-tunnel) tutorial.
 
-Once you start ngrok, note your ngrok ID. You'll use it later on in this guide.
+After you have successfully started ngrok, make sure to note your ngrok ID. You will use it later.
 
 ## Create an app profile
 
-Create a draft app to generate app API account credentials. 
+As a next step, you need to create a draft app to generate app API account credentials. 
+
+To develop and test the app, you need to install it on your [sandbox store](#configure-accounts). 
 
-To develop and test the app, you'll install it on your [sandbox store](#configure-accounts). 
+**To create a draft app**
 
-1. Sign in to the Dev Portal. Then follow our directions to [create an app](/api-docs/apps/guide/developer-portal#create-an-app?source=subscription-foundation). Enter the following values on the **Technical** tab.
+1. Sign in to the Dev Portal. Then follow the directions to [create an app](/api-docs/apps/guide/developer-portal#create-an-app?source=subscription-foundation). Enter the following values on the **Technical** tab.
 
-2. In the **Callback URLs** section, supply the following URLs, replacing `{ngrok_id}` with your ngrok ID from the preceding section.
+2. In the **Callback URLs** section, supply the following URLs, replacing `{ngrok_id}` with your ngrok ID which you noted in an earlier step.
```

Comment:
```suggestion
2. In the **Callback URLs** section, supply the following URLs. Replace `{ngrok_id}` with the ngrok ID you noted in an earlier step.
```
General grammar soapbox items: 
* This kind of `which` construction is common, but isn't grammatically correct. 
* It's clearer and stronger to avoid linking phrases together with present progressives. We want to keep verbs as close to simple present as the grammatical situation allows.

The ngrok ID can (and will) change, so best to avoid possessive articles lest devs get attached and feel slighted when they have to edit the app profile later.